app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="fifty">50dp</dimen>
```

Comment:
mention here dp and sp in name like fifty_dp to reduce the misplace of sp instead of dp